transformers/uws/server.js

```diff
@@ -76,6 +76,16 @@ module.exports = function server() {
     const secKey = req.headers['sec-websocket-key'];
 
     if (secKey && secKey.length === 24) {
+      //
+      // Destroy the socket if it has already been sent a FIN packet.
+      //
+      if (!socket.readable || !socket.writable) return socket.destroy();
```

Comment:
This is probably the only thing that we need. I think adding an error listener is pointless as the socket is destroyed in the same tick.